src/containers/shared/flags.jsx

```diff
@@ -0,0 +1,12 @@
+/*
+These are the supported flags for each api call. Please add them as necessary.
+ */
+
+const flags = {
```

Comment:
`ACCOUNT_FLAGS` in `rippled/lib/utils.js` has the same info.

Reply:
The account flags in utils dont work because theyre an int to string lookup whereas these flags are string to int so this file is still needed unless I'm missing something?

Reply:
I will take a look where it is used.